src/translations/nbTranslations.json

```
@@ -453,6 +454,7 @@
       "update_orcid": "ORCID-kobling ble oppdatert",
       "update_person": "Person ble oppdatert",
       "update_project": "Prosjektet ble oppdatert",
+      "updated_promoted_publication": "Favorittmarkering ble oppdatert",
```

pirk for alfabetisering og matche de andre
```suggestion
      "update_promoted_publication": "Favorittmarkering ble oppdatert",
```